app/views/blog/posts/2011-07-11-reset.html

```diff
@@ -1,514 +0,0 @@
----
```

Comment:
I must confess I really liked this blog post, help me a lot to better understand git reset in the beginning :)

Reply:
It has been more or less moved to the [book](https://github.com/progit/progit2/blob/master/book/07-git-tools/sections/reset.asc)